extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts

```diff
@@ -169,6 +169,13 @@ export class NumericInputValidationService {
             var x = rule.inputs.x as number;
             var tol = rule.inputs.tol as number;
             setLowerAndUpperBounds(range, x - tol, x + tol, true, true);
+            if (tol <= 0) {
```

Comment:
Is there a test for `tol>0`?

Reply:
The unit test directly below the one I've added uses `tol > 0`.